leakcanary-android-core/src/main/java/leakcanary/LeakCanary.kt

```diff
@@ -157,9 +157,24 @@ object LeakCanary {
    */
   @Volatile
   var config: Config = if (AppWatcher.isInstalled) Config() else InternalLeakCanary.noInstallConfig
-    set(value) {
-      field = value
-      SharkLog.d { "Updated LeakCanary.config to $value" }
+    set(newConfig) {
+      val previousConfig = field
+      field = newConfig
+      SharkLog.d {
```

Comment:
A small comment explaining the following blob of code might improve readability here. Something like `// Log the new diff between the old config and new`